ExampleApp/DwollaBroker.cs

```diff
@@ -32,19 +32,34 @@ public class DwollaBroker
         public async Task<RootResponse> GetRootAsync() =>
             (await GetAsync<RootResponse>(new Uri(_client.ApiBaseAddress))).Content;
 
-        public async Task<Uri> CreateCustomerAsync(Uri uri, string firstName, string lastName, string email)
+        public async Task<Uri> CreateCustomerAsync(Uri uri, string firstName, string lastName, string email, DateTime? dateOfBirth = null)
```

Comment:
Dob is only required for VCRs but so are other fields that are shown here. I'd rather not add it to the example and cause confusion.

Reply:
I think I added this to test out some details of verification. Removed.